src/mesh/index_derivs.cxx

```diff
@@ -419,7 +421,7 @@ static DiffNameLookup DiffNameTable[] = { {DIFF_U1, "U1", "First order upwinding
 					  {DIFF_W2, "W2", "Second order WENO"},
 					  {DIFF_W3, "W3", "Third order WENO"},
 					  {DIFF_C4, "C4", "Fourth order central"},
-					  {DIFF_U4, "U4", "Fourth order upwinding"},
+					  {DIFF_U3, "U3", "Third order upwinding"},
```

Comment:
We should keep the `"U4"` name as well, but make it a synonym for `"U3"` till after 4.2